cypress/integration/v2/blocks/pricing-box.spec.js

```diff
@@ -53,3 +59,317 @@ function switchDesign() {
+	} )
+	assertAligns( 'Align', '.ugb-inner-block', { viewport } )
+
+	// Container Tab
+	cy.collapse( 'Container' )
+
+	assertContainer( '.ugb-pricing-box__item', { viewport }, 'column%sBackgroundMediaUrl' )
+
+	range( 1, 4 ).forEach( idx => {
+		cy.setBlockAttribute( {
+			[ `image${ idx }Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		} )
+	} )
+
+	// Spacing Tab
+	cy.collapse( 'Spacing' )
+
+	cy.adjust( 'Paddings', 30, { viewport } ).assertComputedStyle( {
+		'.ugb-pricing-box__item': {
+			'padding-top': '30px',
+			'padding-bottom': '30px',
+			'padding-right': '30px',
+			'padding-left': '30px',
+		},
+	} )
+	cy.resetStyle( 'Paddings' )
+	cy.adjust( 'Paddings', 5, { viewport, unit: 'em' } ).assertComputedStyle( {
+		'.ugb-pricing-box__item': {
+			'padding-top': '5em',
+			'padding-bottom': '5em',
+			'padding-right': '5em',
+			'padding-left': '5em',
+		},
+	} )
+	cy.resetStyle( 'Paddings' )
+	cy.adjust( 'Paddings', 25, { viewport, unit: '%' } ).assertComputedStyle( {
+		'.ugb-pricing-box__item': {
+			'padding-top': '25%',
+			'padding-bottom': '25%',
+			'padding-right': '25%',
+			'padding-left': '25%',
+		},
+	} )
+
+	cy.adjust( 'Image', 13, { viewport } )
+	cy.adjust( 'Title', 18, { viewport } )
+	cy.adjust( 'Price', 23, { viewport } )
+	cy.adjust( 'Sub Price', 28, { viewport } )
+	cy.adjust( 'Button', 33, { viewport } )
+	cy.adjust( 'Description', 38, { viewport } )
+		.assertComputedStyle( {
+			'.ugb-pricing-box__image': {
+				'margin-bottom': '13px',
+			},
+			'.ugb-pricing-box__title': {
+				'margin-bottom': '18px',
+			},
+			'.ugb-pricing-box__price-wrapper': {
+				'margin-bottom': '23px',
+			},
+			'.ugb-pricing-box__subprice': {
+				'margin-bottom': '28px',
+			},
+			'.ugb-pricing-box__button': {
+				'margin-bottom': '33px',
+			},
+			'.ugb-pricing-box__description': {
+				'margin-bottom': '38px',
+			},
+		} )
+
+	// Image Tab
+	cy.collapse( 'Image' )
+
+	desktopOnly( () => {
+		cy.adjust( 'Shape', {
+			label: 'Blob 1',
+			value: 'blob1',
+		} )
+		cy.adjust( 'Flip Shape Horizontally', true )
+		cy.adjust( 'Flip Shape Vertically', true )
+		cy.adjust( 'Stretch Shape Mask', true ).assertClassName( 'img.ugb-img--shape', 'ugb-image--shape-stretch' )
+
+		// We won't be able to assert image size for now since it requires server handling.
+	} )
+
+	cy.adjust( 'Image Width', 300, { viewport } )
+	cy.adjust( 'Force square image', true, { viewport } ).assertComputedStyle( {
+		'.ugb-img': {
+			'width': '300px',
+			'height': '300px',
+		},
+	} )
+	// TO DO: Make assert align for image
+
+	// Title, Price, Price Prefix, Price Suffix, Sub Price and Description Tabs
+	const typographyAssertions = [ 'title', 'price', 'price-prefix', 'price-suffix', 'subprice', 'description' ]
+	typographyAssertions.forEach( typographyAssertion => {
+		// Handle Sub Price label
+		const label = typographyAssertion === 'subprice' ? 'Sub Price' : typographyAssertion.split( '-' ).map( word => startCase( word ) ).join( ' ' )
```

Comment:
`startCase` function can already handle kebab casing. No need to split the words